includes/admin/register-settings.php

```diff
@@ -234,6 +234,13 @@ function wzkb_get_registered_settings() {
 					'type'             => 'text',
 					'options'          => 'kb-tags',
 				),
+				'clickable_section'     => array(
+					'id'               => 'clickable_section',
+					'name'             => esc_html__( 'Section title clickable', 'knowledgebase' ),
+					'desc'             => esc_html__( 'Allow Section title to be a link to his specific section page.', 'knowledgebase' ),
```

Comment:
If selected, the title of each section of the knowledgebase will be linked to its own page